src/ajax/apache.clj

```diff
@@ -62,6 +62,12 @@
   (-status-text [this]
     (let [^HttpResponse response (:response this)]
       (-> response .getStatusLine .getReasonPhrase)))
+  (-get-all-headers [this]
```

Comment:
One of these days someone's going to have duplicate headers. And I'll lay odds I'll receive an issue. :)